pylmod/gradebook.py

```diff
@@ -948,3 +1024,152 @@ def spreadsheet2gradebook(
             csv_reader, email_field, non_assignment_fields
         )
         return response
+
+    def get_staff(self, gradebook_id, simple=False):
+        """Get staff list for gradebook.
+
+        Get staff list for the gradebook specified.
+
+        If simple=True, return a list of dictionaries, one dictionary
```

Comment:
Should we add personId and call it a day? (I'm assuming personId is a parameter for other API calls, the same way the API uses courseid and gradebookid)